libs/sdk-ui-dashboard/src/presentation/topBar/title/RenderModeAwareTitle.tsx

```diff
@@ -0,0 +1,13 @@
+// (C) 2022 GoodData Corporation
+
+import { DefaultTitle } from "./DefaultTitle";
+import { EditableTitle } from "./EditableTitle";
+import { renderModeAware } from "../../componentDefinition";
+
+/**
+ * @internal
+ */
+export const RenderModeAwareTitle = renderModeAware({
+    view: DefaultTitle,
+    edit: EditableTitle,
```

Comment:
I think that this task is also about calling the relevant command, when title is changed. (`calls the RenameDashboard command on blur or Enter press`)

Reply:
Also, please check https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like (at least, you should mention the ticket number in the commit message)

Reply:
> I think that this task is also about calling the relevant command, when title is changed. (`calls the RenameDashboard command on blur or Enter press` https://jira.intgdc.com/browse/RAIL-4105)

It is already linked and called here https://github.com/gooddata/gooddata-ui-sdk/blob/2823b00b889f2624366ed6140d4c38e568a7b1ca/libs/sdk-ui-dashboard/src/presentation/dashboard/DashboardHeader/DashboardHeader.tsx#L259

Reply:
> Also, please check https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like (at least, you should mention the ticket number in the commit message)

I fixed that

Reply:
Oh, I missed that as it's not part of this PR. Thanks for the explanation.
Thank you.